src/views/Auth/Login.jsx

```diff
@@ -37,12 +43,16 @@ export default function Login() {
           id="email"
           name="email"
           type="email"
```

Comment:
We should set the value of each of these inputs to their related state so that React has full control over them